src/transformers/pipelines/audio_utils.py

```diff
@@ -51,6 +55,7 @@ def ffmpeg_microphone(
     chunk_length_s: float,
     format_for_conversion: str = "f32le",
     ffmpeg_input_device: Optional[str] = None,
+    ffmpeg_additional_args: Optional[list] = [],
```

Comment:
No mutables as default arguments please! 

The right way to do this is have this unset to `None` by default and then default to an empty list if unset 

```suggestion
    ffmpeg_additional_args: Optional[List[str]] = None,
```

then later on

```py
    ffmpeg_additional_args = [] if ffmpeg_additional_args is None else ffmpeg_additional_args
```